packages/neuron-ui/src/locales/en.json

```diff
@@ -272,7 +271,10 @@
       "rpc-url-should-have-protocol": "The RPC URL should start with http(s)://",
       "rpc-url-should-have-no-whitespaces": "The RPC URL should have no whitespaces",
       "is-required": "{{field}} is required",
-      "is-used": "{{field}} is used"
+      "is-used": "{{field}} is used",
+      "invalid-address": "{address}} is an invalid address",
+      "invalid-checksum": "{{address}} has an invalid checksum",
+      "address-length-error": "{{address}} has an invalid length"
```

Comment:
Same as above, although we have different kinds of invalid reasons, show something like "The address you input is invalid" or so is more friendly and generically enough.